src/mapValues.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Maps values of `object` and keeps the same keys.
+ * @param object the object to map
+ * @param fn the mapping function
+ * @signature
+ *    R.mapValues(object, fn)
+ * @example
+ *    R.mapValues({a: 1, b: 2}, (value, key) => value + key) // => {a: '1a', b: '2b'}
+ * @data_first
+ * @category Object
+ */
+export function mapValues<T extends object, S>(
+  object: T,
+  fn: (value: T[keyof T], key: keyof T) => S
+): Record<keyof T, S>;
+
+/**
+ * Maps values of `object` and keeps the same keys.
+ * @param fn the mapping function
+ * @signature
+ *    R.mapKeys(fn)(object)
+ * @example
+ *    R.pipe({a: 1, b: 2}, R.mapValues((value, key) => value + key)) // => {a: '1a', b: '2b'}
+ * @data_last
+ * @category Object
+ */
+export function mapValues<T extends object, S>(
+  fn: (key: keyof T, value: T[keyof T]) => S
```

Comment:
I made a terrible mistake here: The function should actually take the `value` first and the `key` as second parameter, just like [here](https://github.com/remeda/remeda/pull/86/files#diff-e107e2315272076f1de4296bd17beb11R14).

The tests pass because the "runtime" function does it right :/

I'll make a PR to fix this immediately.

Reply:
here is the PR: https://github.com/remeda/remeda/pull/93 🙈